README.md

```diff
@@ -19,6 +19,7 @@
 ### Matchers
 
 - [jest-extended](https://www.github.com/jest-community/jest-extended) Adds additional matchers to core API making it easy to test everything.
+- [expect-more](https://github.com/JamieMason/expect-more/tree/master/packages/expect-more-jest)
```

Comment:
@JamieMason what do you want as text here?

Reply:
Thanks @SimenB. "A huge library of test matchers for a range of common use-cases" from the readme would be great, thank you.

Reply:
Done!